redux-worker-middleware.d.ts

```diff
@@ -0,0 +1,7 @@
+import { Middleware } from "redux";
+
+declare function createWorkerMiddleware(
+  worker: Worker,
+  meta?: string
```

Comment:
Would you mind renaming it to `workerName`? Also, the type should probably be:
```
workerName?: string | boolean,
```

Reply:
✅